src/components/graph-vis/filters.js

```diff
@@ -16,6 +18,21 @@ class Filters extends React.Component {
   render() {
     const store = this.context;
     return (<Grid>
+      { /* dataset specific */}
+      {store.yearRange && <RangeSliderControlComponent
+        value={store.year}
```

Comment:
```suggestion
        value={store.year.slice()}
```

the Slider component internally calls `.sort` on its given values. Thus, the mobx warning.